dags/nf-validate-test.py

```diff
@@ -49,8 +33,7 @@ def launch_tower_workflow(tower_utils: TowerUtils, workspace_id: str):
             workspace_secrets=["SYNAPSE_AUTH_TOKEN"],
         )
 
-    tower_utils = open_tower_workspace()
-    launch_tower_workflow(tower_utils["tower_utils"], "4034472240746")
+    launch_tower_workflow("4034472240746")
```

Comment:
My instinct was to ask whether we can make this a DAG parameter, but we can wait until we have a Nextflow Tower hook that supports workspaces being encoded in the connection URI.

Reply:
I agree on waiting